src/test/java/com/google/cloud/spanner/pgadapter/JdbcMockServerTest.java

```diff
@@ -138,6 +138,39 @@ public void testSelectCurrentSchema() throws SQLException {
     assertEquals(0, mockSpanner.countRequestsOfType(ExecuteSqlRequest.class));
   }
 
+  @Test
+  public void testSelectCurrentDatabase() throws SQLException {
+    String sql = "SELECT current_database";
```

Comment:
Have you tested with the scenarios like `Select current_database()`, `SeLect cUrrent_DATabase`, etc? Should we add such tests here also?

Reply:
No, they are intentionally not all supported (see above)